src/includes/grouping/in-app/apple.mdx

```diff
@@ -1,6 +1,6 @@
 The sentry-cocoa SDK always marks public frameworks such as  UIKitCore, CoreFoundation, GraphicsServices, and so forth as `not inApp`.
-All private frameworks inside the application bundle are marked as `inApp`. For instance, the following marks frames from libdispatch
-starting with `_dispatch_` as `inApp`.
+All private frameworks and the main executable inside the application bundle are marked as `inApp`. The following marks
+frames from libdispatch starting with `_dispatch_` as `inApp`.
```

Comment:
_The following marks starting with `_dispatch_` as `inApp`.	frames from_

this reads a bit weird for me, what's about.

The frames from libdispatch starting with `_dispatch_` are marked as `inApp`

Reply:
Hmm, I try to avoid the passive voice as much as I can. For me, it's fine.